src/dataAccessLayer.ts

```diff
@@ -32,18 +35,46 @@ export class DataAccessLayer {
         return await this.blockchainRepository.getBlockchainAsync();
     }
 
+    public async getBlocksByPeriodAsync(beginTimestamp: number, endTimestamp: number): Promise<Block[]> {
+        return await this.blockchainRepository.getBlocksByPeriodAsync(beginTimestamp, endTimestamp);
+    }
+
+    public async getBlocksByDateAsync(timestamp: number): Promise<Block[]> {
```

Comment:
The method indicates it will return multiple blocks by date, but a timestamp needs te passed instead?

Reply:
A timestamp is probably too specific but currently the only time value saved in a BlockHeader. Would the best alternative be to remove this function and use getBlocksByDatePeriod instead?